taskcluster/ci/lint/kind.yml

```diff
@@ -16,15 +16,14 @@ job-defaults:
     docker-image: { in-tree: linux }
     max-run-time: 1800
   run:
-    using: run-task
-    cwd: "{checkout}"
+    using: gradlew
 
 jobs:
   detekt:
-    description: "detekt"
+    description: "Running detekt over all modules"
     run:
-      command: ./gradlew --no-daemon clean detekt
+      gradlew: [detekt]
   ktlint:
-    description: "ktlint"
+    description: "Running ktlint over all modules"
     run:
-      command: ./gradlew --no-daemon clean ktlint
+      gradlew: [ktlint]
```

Comment:
@JohanLorenzo Nit: we're losing the `--no-daemon clean` arguments along the way with this migration; I checked and it seems that's the case in other mobile projects too. Just wanted to double check whether this was intended or not.

Reply:
I think it's okay in this case because the docker image doesn't contain intermediary build artifacts anymore. So, I don't think there's anything to clean yet.